packages/xhr-mock/src/proxy.browser.ts

```diff
@@ -22,7 +22,8 @@ export default function(
     const xhr: XMLHttpRequest = new XHRMock.RealXMLHttpRequest();
 
     // TODO: reject with the correct type of error
```

Comment:
Can probably get rid of the `TODO:` now? 😄

Reply:
Will do